src/Documentation/UserManual/DeviceOpticalMarkerTracker.dox

```diff
@@ -0,0 +1,86 @@
+/*!
+\page DeviceOpticalMarkerTracker Optical Marker Tracker
+
+\section DeviceOpticalMarkerTrackerSupportedHwDevices Supported hardware devices
+
+- Any external webcam or laptop camera (forward or rear facing).
+
+\section DeviceOpticalMarkerTrackerSupportedPlatforms Included in packages
+
+- \ref PackageWin32
+- \ref PackageWin64
+- \ref PackageInterson
+- \ref PackageBk
+- \ref PackageStealthLink
+- \ref PackageThorLabs
+
+\section DeviceOpticalMarkerTrackerCalibration Calibration
+
+- Print the <a href="https://www.assembla.com/spaces/plus/documents/cd0kq6t7ir577cdmr6QqzO/download/cd0kq6t7ir577cdmr6QqzO">aruco calibration chessboard</a> and measure the side length of the markers in meters.
```

Comment:
Refer to the file in the repository instead (on github or assembla). Assembla files are not that reliable.